tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -1114,6 +1122,12 @@ static __always_inline int should_trace()
     {
         return 0;
     }
+
+    if (!equality_filter_matches(CONFIG_PROC_TREE_FILTER, &process_tree_map, &context.pid))
```

Comment:
If you are saving tgids in the map - why do you campre to pid?

Reply:
In the `struct event_context` definition you have the 'pid' field commented with `PID as in the userspace term` which appears to be accurate int `init_context`

Reply:
Yes, you are right